openslides/core/static/js/core/projector.js

```diff
@@ -57,6 +57,60 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
     }
 ])
 
+// Projector Container Controller
+.controller('ProjectorContainerCtrl', [
+    '$scope',
+    'Config',
+    function($scope, Config) {
+        // watch for changes in Config
+        $scope.$watch(function () {
+            return Config.lastModified();
+        }, function () {
+            var split = Config.get('projector_resolution').value.split('_');
+            $scope.projectorWidth = parseInt(split[0]);
+            $scope.projectorHeight = parseInt(split[1]);
+            $scope.recalculateIframe();
```

Comment:
Try to recalculate only if the special config is changed - not if any config changed.